docs/.vuepress/config.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
     "/": {
       lang: "en-US",
       title: "Beego",
-      description: "The most easy use framework",
+      description: "The easiest to use framework",
```

Comment:
Do not change this file except this fix.

Reply:
You meant, not to change the config.js file?